ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -122,9 +122,9 @@ public static void setUp() {
     @AfterAll
     public static void tearDown() {
         try {
-            queryClient.executeToJsonResult(databaseName, String.format(".drop table %s ifexists", tableName));
             ingestClient.close();
```

Comment:
Why the order change?
If the close() fails for one the above, it would leave the table hanging